src/components/ChangePlan/Card/Card.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+
+const Card = ({ data }) => {
+
+  return (
+    <div class="dp-card">
+      <h3>{data.title}</h3>
+      <p>{data.description}</p>
+      <p>{data.descriptionPlan}</p>
+      <div class="dp-price" data-start={data.price.initialText}>
```

Comment:
What is the meaning of _initialText_ or _start_, is it possible thinking in a more descriptive name?

Reply:
Regarding the names, what do you think about something like this:

```suggestion
      <div class="dp-price" data-price-prefix={data.price-prefix}>
```

I think that the prefix is related to the price but it is not part of the price. Am I missing something?

Reply:
yes, its 'since' and 'per month'